### ISSUE TITLE
Amber not working on Mac

### ISSUE THREAD

**Issue by 11reed** [amberframework/amber#1044]
##### I am just trying to get Amber working on Mac.

I get this error when I run `amber db create migrate`:

![alt text](https://i.imgur.com/PCIQxb4.png "Logo Title Text 1")

I also get this error when I run `amber watch`:

![alt text](https://i.imgur.com/x7YmCOT.png "Logo Title Text 1")

### Additional Information

I've tried uninstalling & reinstalling Amber and Crystal with brew, & no fixes.
I have `openssl` installed and `pkg-config`.
Let me know if I need to put more info.

**Comment by 11reed** [comment]
Nope, I still get the same error. Maybe I should try reinstalling everything again?

**Comment by drujensen** [comment]
does `brew doctor` show any issues?

**Comment by 11reed** [comment]
I didn't have `/usr/local/sbin` so I created & have ownership. I tried running `amber watch` again and `crystal src/test-amber.cr` didn't work. That was the only issue in `brew doctor` besides some unlinked things but not really relevant to this.

**Comment by drujensen** [comment]
could it be related to this? https://github.com/crystal-lang/crystal/issues/3067

**Comment by 11reed** [comment]
Nope, it must be something else.

**Comment by drujensen** [comment]
just to confirm, mysql is installed and running?  you can connect using `mysql -u root`?

**Comment by 11reed** [comment]
Accidentally closed, but yeah it's running.

**Comment by elorest** [comment]
Which version of macOS are you using? I'd verify that crystal is working and try manually installing amber instead of using brew version.

**Comment by 11reed** [comment]
I'm using Mojave latest release. Tried manually installing amber and I still get the same error. I keep getting `ld: library not found for -lssl (this usually means you need to install the development package for libssl)` even though everything is installed.

**Comment by drujensen** [comment]
try adding this to your `.profile`:
```
export PKG_CONFIG_PATH=/usr/local/opt/openssl/lib/pkgconfig
```
